editor/components/rich-text/format-toolbar/index.js

```diff
@@ -22,7 +23,7 @@ import { filterURLForDisplay } from '../../../utils/url';
 
 const { ESCAPE, LEFT, RIGHT, UP, DOWN, BACKSPACE, ENTER, displayShortcut } = keycodes;
 
-const FORMATTING_CONTROLS = [
+const FORMATTING_CONTROLS = applyFilters( 'editor.richText.formattingControls', [
```

Comment:
For consistency, we should name this filter `editor.RichText.formattingControls`.